src/Extension/CommonMark/Node/Inline/Image.php

```diff
@@ -16,23 +16,29 @@
 
 namespace League\CommonMark\Extension\CommonMark\Node\Inline;
 
-use League\CommonMark\Node\Inline\Text;
-
 class Image extends AbstractWebResource
 {
+    protected string $label;
     protected ?string $title = null;
 
-    public function __construct(string $url, ?string $label = null, ?string $title = null)
+    public function __construct(string $url, ?string $label = '', ?string $title = null)
```

Comment:
I don't think this change is correct.

Reply:
My logic with this change is that, as far as I can tell, it's impossible for an <img> tag created with MD/CM to not have an alt attribute. `![](foo.jpeg)` results in `<img src="foo.jpeg" alt="" />`.